purchases/src/main/kotlin/com/revenuecat/purchases/common/diagnostics/DiagnosticsTracker.kt

```diff
@@ -529,3 +567,10 @@ internal class DiagnosticsTracker(
         diagnosticsDispatcher.enqueue(command = command)
     }
 }
+
+val ProductType.diagnosticsName: String
```

Comment:
this should be `private` right?
```suggestion
private val ProductType.diagnosticsName: String
```